### PR TITLE
Improve contrast of progress bar showing number of failures

### PR DESCRIPTION
**Pull request by Lokeshsri11**
This commit aims to enhance the contrast of the progress bar used to display the number of failures. The previous implementation had low contrast, making it difficult to discern the actual progress. The updated design uses a more prominent color scheme, improving visibility and making it easier to understand the number of failures at a glance. This change improves the overall user experience and ensures that the progress bar is more accessible to all users.

232456699-4f5afdd7-306c-4399-856b-b8725b85bd3d

### CODE REVIEW

**Review by mimi1vx — APPROVED** [review]

**Comment by Martchus** [comment]
The commit message is not very clear. I'd rephrase it as "Improve contrast of progress bar showing number of failures" and also add a brief version of the description you've so far only added as PR description as commit message body.

**Comment by Lokeshsri11** [comment]
@Martchus See the description, is it right now?

**Comment by Martchus** [comment]
I've merged it as https://github.com/os-autoinst/openQA/commit/cbbc51b6f9fdac9a90356b5f49daae73e33c1dfb. Notice that the commit itself contains the updated title/body. If you wanted to update the commit itself yourself you need to use `git commit --amend` (or an interactive rebase it if isn't just about the last commit) and then force-push.

**Comment by Lokeshsri11** [comment]
> No it is not. I was talking about the commit message and not the pull request title. However, I can just change it when merging manually.

okay :)
